ui/src/views/Documentation/Reference/LogsReference.jsx

```diff
@@ -35,16 +35,18 @@ export default class LogsReference extends Component {
         {ref.description && (
```

Comment:
For `{ref.title && <HeaderWithAnchor>{ref.title}</HeaderWithAnchor>}`, when `ref.title` is not defined, let's show `Logs` instead:

```jsx
<HeaderWithAnchor>{ref.title || 'Logs'}</HeaderWithAnchor>
```